### ISSUE TITLE
GUI : FED : Update password validations

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#2220]
### As is
If IPMI is enabled the user is restricted to a password with a maximum of 20 characters. This is due to IPMI restrictions. Our password validation only shows an error if the password field is empty.

### To be
If IPMI is enabled, the user should receive a real-time error message when/if they add a password longer than 20 characters. 

#### Error message
`Password cannot exceed 20 characters when IPMI is enabled`

## UX Flow / Interaction Requirements
- [ ] Feature breakdown

## Final Invision Mockup
- [ ] Add Link

## cURL Commands
- [ ] Add cURL commands

## Notes
- [ ] Any known API workarounds
- [ ] Any server-side functionality WIP

### Browser Tests (Chrome, Firefox, Safari (Mac), Edge (Windows))
- [ ] Tested responsive layout on supported viewport sizes
- [ ] Verified functionality works as expected, e.g. success and failure use cases
- [ ] Resolved any JavaScript errors thrown to the console

### Accessibility Tests
- [ ] Verified markup is semantic
- [ ] No reported issues in DAP

**Comment by derick-montague** [comment]
Added a new blocker which has the same milestone. They will need updated milestones once the work has higher priority.

**Comment by dixsie** [comment]
Still blocked, waiting on #2219. Need to follow up with @joseph-reynolds

**Comment by joseph-reynolds** [comment]
#2219 is completed [edit: for IBM branches] and is no longer a blocker [for IBM branches].  It would be nice to have a real-time warning as they type which says `Password cannot exceed 20 characters when IPMI is enabled`, so this story is still relevant.  The Web app can detect if SOMEUSER is allowed to use the IPMI interface by checking if the "IPMI" value is in the list of AcountTypes which is a property in /redfish/v1/AccountService/Accounts/SOMEUSER.

Note an interim solution currently being implemented is to offer help text for the new password field, which includes guidance that says the password is limited to 20 characters when the user is allowed to use IPMI.
